app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,6 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:networkSecurityConfig="@xml/network_security_config"
```

Comment:
Can you delete this and the associated xml while you're at it? The config currently turns on cleartext traffic which I've been using for testing locally, but probably not appropriate for a template.